src/wp-includes/blocks.php

```diff
@@ -175,11 +175,21 @@ function register_block_script_module_id( $metadata, $field_name, $index = 0 ) {
 	$block_version       = isset( $metadata['version'] ) ? $metadata['version'] : false;
 	$module_version      = isset( $module_asset['version'] ) ? $module_asset['version'] : $block_version;
 
+	$args = array();
+	if (
+		( isset( $metadata['supports']['interactivity'] ) && true === $metadata['supports']['interactivity'] ) ||
+		( isset( $metadata['supports']['interactivity']['interactive'] ) && true === $metadata['supports']['interactivity']['interactive'] )
+	) {
+		// TODO: Add ability for the fetchpriority to be specified in block.json for the viewScriptModule. In wp_default_script_modules() the fetchpriority defaults to low since server-side rendering is employed for core blocks, but there are no guarantees that this is the case for non-core blocks. That said, viewScriptModule entails Interactivity API, following the pattern from core it _should_ be SSR'ed and that is why this is the default for when the block supports interactivity.
```

Comment:
This comment seems slightly misleading:

> viewScriptModule entails Interactivity API

Interactivity API relies on Script Modules, not the other way around. There's no need to use Interactivity API with `viewScriptModule` nor with Script Modules in general.

> following the pattern from core it _should_ be SSR'ed and that is why this is the default for when the block supports interactivity.

Agreed, Interactivity API regions should be server rendered.

Reply:
@sirreal Sorry, just getting to your comment now. 

> Interactivity API relies on Script Modules, not the other way around. There's no need to use Interactivity API with `viewScriptModule` nor with Script Modules in general.

How about this:

```suggestion
		// TODO: Add ability for the fetchpriority to be specified in block.json for the viewScriptModule. In wp_default_script_modules() the fetchpriority defaults to low since server-side rendering is employed for core blocks, and it is a key design principle for the Interactivity API is for blocks to use SSR; fetchpriority=low is the default for when the block supports interactivity, since if SSR is not employed the block is essentially doing it wrong.
```

Reply:
Actually, I'm going to remove the comment entirely in favor of putting it into a dedicated issue: https://github.com/WordPress/gutenberg/issues/71366